src/app/ReadHandler.h

```diff
@@ -372,6 +384,7 @@ class ReadHandler : public Messaging::ExchangeDelegate
      *  @param    keepPersisted             Keep the subscription persisted in storage for later resumption
```

Comment:
Add a note above that if CHIP_CONFIG_PERSIST_SUBSCRIPTIONS is not enabled, keepPersisted argument is ignored

Reply:
This was replaced by what Andrei suggested - an enum class as Close() argument.